### ISSUE TITLE
MusicXML import - OSError: Process didn't complete in time

### ISSUE THREAD

**Issue by fedelibre** [wbsoft/frescobaldi#1105]
I'm trying to import a MusicXML file (extension changed from .mxl to .txt to be able to attach it here):
[debussy-deux-arabesques.txt](https://github.com/wbsoft/frescobaldi/files/2397742/debussy-deux-arabesques.txt)

The .ly file is generated but it's not displayed on Fresco editor pane, while the following error appears:

```
Traceback (most recent call last):
  File "/home/fede/src/frescobaldi/frescobaldi_app/file_import/__init__.py", line 99, in importMusicXML
    self.openMusicxmlDialog()
  File "/home/fede/src/frescobaldi/frescobaldi_app/file_import/__init__.py", line 109, in openMusicxmlDialog
    self.runImport()
  File "/home/fede/src/frescobaldi/frescobaldi_app/file_import/__init__.py", line 158, in runImport
    j = dlg.run_command()
  File "/home/fede/src/frescobaldi/frescobaldi_app/file_import/toly_dialog.py", line 155, in run_command
    raise OSError("Process didn't complete in time")
OSError: Process didn't complete in time
```

**Comment by uliska** [comment]
#1100

**Comment by uliska** [comment]
I can't fully reproduce the problem. I see that the editor doesn't load the file (which should be easy to fix) although it has been created.

I have hardcoded a 3 seconds timeout for external commands like the import (in order to prevent endless loops). Maybe that should be increased or made configurable. In any case I have the impression that you automatically engrave the file and that is also (wrongly) included by the timeout. 

Unfortunately I can't access your attached file (is downloaded as garbage). Maybe post it in a code block?

**Comment by fedelibre** [comment]
Sorry, I didn't check the file, my bad! It's actually a zip file containing an .xml file. And it's pretty long: 60 thousand lines! Try this:
[lg-204746362.txt](https://github.com/wbsoft/frescobaldi/files/2397834/lg-204746362.txt)

I downloaded it from [musescore.org](https://musescore.com/openscore/scores/4788704) (login required).

**Comment by fedelibre** [comment]
Another weird issue with midi2ly. When I import a very simple midi file generated by lilypond, a new .ly empty file is generated and its name has a kind of progressive number. Let me explain better with my example:

```
$ ls -lh issue-104*.ly
-rw-rw-r-- 1 fede fede 247 16 lug 18.11 issue-1042.ly
-rw-rw-r-- 1 fede fede 908 19 set 18.04 issue-1042-midi.ly
-rw-rw-r-- 1 fede fede   0 19 set 17.48 issue-1043.ly
-rw-rw-r-- 1 fede fede   0 19 set 17.49 issue-1044.ly
-rw-rw-r-- 1 fede fede   0 19 set 17.53 issue-1045.ly
-rw-rw-r-- 1 fede fede   0 19 set 17.53 issue-1046.ly
-rw-rw-r-- 1 fede fede   0 19 set 18.04 issue-1047.ly

```

where:

- issue-1042.ly is the original ly file which generated the midi
- issue-1042-midi.ly is the file correctly imported by midi2ly on the command line
- issue-1043.ly and following files are generated by Frescobaldi when importing from MIDI

**Comment by uliska** [comment]
OK, thanks for the file with which I can now reproduce the behaviour. I'll have to investigate ...

**Comment by uliska** [comment]
Will be automatically fixed by #1109
